### ISSUE TITLE
Connection can't be deleted from connection details

### ISSUE THREAD

**Issue by mcada** [syndesisio/syndesis-ui#1179]
I think it should be possible to delete connection if you are in connection details. It is possible to delete integration from integration details.

**Comment by gashcrumb** [comment]
+1 

@dongniwang If we add this would it make sense to have it in a similar location as the integration details?

**Comment by dongniwang** [comment]
Agreed that we should have the ability to delete a connection from the connection details page. 

But I think the delete button should probably be placed at the breadcrumb/toolbar level. 

Integration details page is being reworked at the moment, will need to align button placement for these two pages.
